content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,151 @@
+- Process Modelling can model complex processes that require discussions between engineers and SMEs and help generate a backlog of work
+- Process modelling can be a quick way to visualize an algorithmic flow or pipeline for use in architectural diagrams and other modelling tools
+- Process Modelling can help compare/hypothesise processes to validate proposed solutions
+when:
+- When discovering a high level business process
+- When discovering a pipeline or workflow
+- When validating process changes that might help solve a problem
+what:
+- "Large wall or digital collaboration space like [Miro](https://miro.com/)"
+- 5+ different colored sticky notes (when using this in conjunction with EventStorming and/or BORIS + SNAPe keep consistency in colors)
+- Sharpies
+
+---
+## How to Use this Method
+This is formally a grammar/notation for modelling Processes and Software, that has evolved from the application of [EventStorming](https://www.eventstorming.com/) by its founder Alberto Brandolini. 
+Traditionally it was indistinguishable from `EventStorming`, however its evolution has lead to Process Modelling becoming its own phase in the `EventStorming` methodology.
+
+While the formal method is best used for identifying large complex high level processes, the general principals and grammar of Process Modelling have been adopted successfully to discover or validate workflows and pipelines at much smaller scales and through existing systems.
+
+The output from these workshops can be used to discover, validate or design a system that supports an end to end business process. The information can be used as an algorithmic model of the problem space, and other forms of design can be derived from it; such as an Event + Domain Driven microservice architecture, a simple Pipeline or even design a `CQRS` and Event Sourcing pattern. Typically there will be a follow on exercise to rationalize the information and transform it from an algorithmic model to a more suitable architectural model.
+
+#### Sample Agenda & Prompts - 
+1. Arrange the board:
+     - Label the board to identify a name for the process being explored
+     - From the main storming board, identify a selection of key events that describe an end to end process within the larger system and copy them to another board. 
+       - Preconditions: Identify events which need to have happened before this process can begin and put them in a grayed area on the left with the title Preconditions
+       - Postconditions: Identify at least one suitable event which describes the optimal outcome of the process
+       - Arrange any remaining events in a chronological order from left to right between the preconditions and optimal outcomes. Do not worry about accuracy, as the events are narrated, be prepared to change, add or remove events
+     - Add a grammar guide as shown  ![This is the placeholder for process modelling grammar](/images/practices/event-storming-process-modelling/process-modelling-grammar.png)
```

Comment:
It would probably make sense here to explain each colour or point to another article doing so.

In comparison with the Design-level-ES exercise, you left a few things out. Is it on purpose?

It would be useful to share a miro template for this as well.